src/utils/searchUtils/searchUtils.ts

```diff
@@ -191,3 +192,12 @@ export function isValidGateway(gateway: string) {
 export function isValidURL(url: string) {
   return url ? URL_REGEX.test(url) : false;
 }
+
+export function isMaxLeaseDuration(duration: number | string) {
+  return (
+    (duration &&
+      typeof duration === 'number' &&
+      duration >= MAX_LEASE_DURATION) ||
+    duration === 'Indefinite'
```

Comment:
We use the same String on `src/components/forms/DomainSettings/DomainSettings.tsx`. Consider creating a new const